docs/memestra-cache.rst

```diff
@@ -0,0 +1,34 @@
+Memestra-cache
+==============
+
+To handle the cache, memestra installs a tool named ``memestra-cache``.
+
+Memestra's caching infrastructure is a file-based cache, located in ~/.memestra (RW) and in memestra/cache (RO). The key is a hash of the file content and the value is deprecation information, generator used, etc.
```

Comment:
`~/.memestra` (formating). Same for `memestra/cache` which should probably `<prefix>/memestra/cache`